ruby/vampires.rb

```diff
@@ -1,4 +1,4 @@
-puts "How many employees will be processed?"
+puts "How many employees would you like to process?"
 applicants = gets.chomp
 
 current_year = 2016

```

Comment:
Thank you for the very helpful feedback. After I finish week 5 work I'll revisit it and try all that.